src/main/java/edu/umich/tl/CourseDelete.java

```diff
@@ -66,16 +80,45 @@ private static void readPropertiesFromFile(String[] args) {
 		String propFileLocation;
 		if(args.length == 0) {
 			M_log.error("Command line arguments are not provided");
-			System.exit(0);
+			System.exit(1);
```

Comment:
Document system exit values / meaning at top of file (if you haven't already done that).